docs/deployment/universal-profile.md

```diff
@@ -75,8 +114,8 @@ const myUniversalProfileData = {
       width: 500,
       height: 500,
       hashFunction: 'keccak256(bytes)',
-      hash: '0xfdafad027ecfe57eb4ad047b938805d1dec209d6e9f960fc320d7b9b11cbed14', // bytes32 hex string of the image hash
-      url: 'ipfs://QmPLqMFHxiUgYAom3Zg4SiwoxDaFcZpHXpCmiDzxrtjSGp',
+      hash: '0xfdafad027ecfe57eb4ad044b938805d1dec209d6e9f960fc320d7b9b11cced14', // bytes32 hex string of the image hash
```

Comment:
How about adding a quick link to this website below?
https://emn178.github.io/online-tools/keccak_256_checksum.html

With a one sentence stating something like _"you can obtain the hash digest of your image via keccak256, using this online keccak256 file checksum tool"_